baseapp/baseapp.go

```diff
@@ -584,7 +585,7 @@ func (app *BaseApp) cacheTxContext(ctx sdk.Context, txBytes []byte) (sdk.Context
 // Note, gas execution info is always returned. A reference to a Result is
 // returned if the tx does not run out of gas and if all the messages are valid
 // and execute successfully. An error is returned otherwise.
-func (app *BaseApp) runTx(mode runTxMode, txBytes []byte) (gInfo sdk.GasInfo, result *sdk.Result, anteEvents []abci.Event, err error) {
+func (app *BaseApp) runTx(mode runTxMode, txBytes []byte) (gInfo sdk.GasInfo, result *sdk.Result, anteEvents []abci.Event, priority int64, err error) {
```

Comment:
What do you think about adding `Priority int64` to the `sdk.Result` type instead? I feels wrong to just keep returning variables here...

Reply:
That was my first idea too yeah. But priority is set in antehandlers, and at that point there's no `sdk.Result` yet

Reply:
Yes, that's fine because you can just set it on `Result` later [here](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/baseapp/baseapp.go#L705)

Reply:
I guess my reserve was: is this conceptually correct? `sdk.Result` imo is more a msg execution result, which is separate from priority/CheckTx stuff.

But if you feel strongly about putting priority in Result, let's do this.